Library/Homebrew/extend/ENV/super.rb

```diff
@@ -307,7 +307,7 @@ def cxx11
     when "clang"
       append "HOMEBREW_CCCFG", "x", ""
       append "HOMEBREW_CCCFG", "g", ""
-    when /gcc-(4\.(8|9)|5)/
+    when /gcc-(4\.(8|9)|5|6)/
```

Comment:
Can you replace `5|6` with `[5-6]` to make it easier to extend this in the future?